slices/debianutils.yaml

```diff
@@ -0,0 +1,30 @@
+package: debianutils
+
+essential:
+  - debianutils_copyright
+
+slices:
+  which:
+    essential:
+      - base-files_bin  # The Shebang in which.debianutils references /bin/sh
+      - dash_bins
+    contents:
+      # Note: This symlink should have a prefer: gnu-which if both are present
+      /usr/bin/which: {symlink: which.debianutils}
+      /usr/bin/which.debianutils:
+
+  ischroot:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/ischroot:
```

Comment:
`ldd` of `ischroot` looks fine, but on a hunch i tried `strings ischroot | grep "lib[^\.]*\.so"` and got a reference to `libfakechroot.so` from [fakechroot](https://manpages.debian.org/testing/fakechroot/fakechroot.1.en.html) deb. for reference, the mention is in `xxd ischroot | grep -B30 -A10 libfakechroot`. i would not add it as a dep but just an interesting case to maybe investigate at some point. :)